Consul/Health.cs

```diff
@@ -45,6 +45,21 @@ public class HealthStatus : IEquatable<HealthStatus>
 
         public static HealthStatus Any { get; } = new HealthStatus() { Status = "any" };
 
+        public static HealthStatus Parse(string status)
+        {
+            switch (status)
```

Comment:
I just realised that the switch statement in the lines 93 to 103 can be replaced now by the `HealthStatus.Parse(status);` .